lib/spark/build.rb

```diff
@@ -1,10 +1,10 @@
 module Spark
   module Build
 
-    DEFAULT_SCALA_VERSION  = '2.10.4'
-    DEFAULT_CORE_VERSION   = '2.10'
-    DEFAULT_SPARK_VERSION  = '1.5.0'
-    DEFAULT_HADOOP_VERSION = '1.0.4'
+    DEFAULT_SCALA_VERSION        = '2.10.4'
+    DEFAULT_SPARK_CORE_VERSION   = '2.10'
+    DEFAULT_SPARK_VERSION        = '1.5.0'
+    DEFAULT_HADOOP_VERSION       = '2.7.1'
```

Comment:
Hadoop version could be changed via command line. I had many problems with Hadopp > 2. Version 1 was "always stable".